Bash_scripts/HW28/Jenkinsfile

```diff
@@ -0,0 +1,48 @@
+pipeline {
+  agent any
+  stages {
+    stage('Shell script 0') {
+      steps {
+        sh '''#!/bin/bash
+
+cd $PATHTO
+SHELL_SCRIPTS=$(find . -name \'*.sh\')
+
+for var in $SHELL_SCRIPTS ; do
+shellcheck "$var"
+done'''
+      }
+    }    
+    stage('Shell script 1') {
+      steps {
+        sh '''#!/bin/bash
+
+DOCKERFILE=$(find "$PATHTO" -name \'Dockerfile\')
+
+for var in $DOCKERFILE ; do
+dockerfile_lint  -f "$var"
+done'''
```

Comment:
Better to move `'''` to a new line,  to make your code more clear

Reply:
fixed